tests/test_kitsune.py

```diff
@@ -211,6 +216,18 @@ def test_fetch_empty(self):
 
         self.assertEqual(len(questions), 0)
 
+    @httpretty.activate
```

Comment:
This test is much better. The server will fail but the backend will continue fetching items.